packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -592,11 +566,11 @@ const DatePickerInput = (props) => {
       return;
     }
     // increment current valid date by 1 day, if not valid date set date to tomorrow instead
```

Comment:
This comment can be removed since you are incrementing date based on input field.

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1999/commits/7b61de3934e318f97325758b279207800dc84d9e